syncer/data_fetch.go

```diff
@@ -13,6 +14,8 @@ import (
 	"github.com/spacemeshos/go-spacemesh/rand"
 )
 
+const pollTimeOut = time.Second * 30
```

Comment:
could you please use 30 * time.Second, it is common in golang to write number before duration type